udata_hydra/cli.py

```diff
@@ -261,7 +265,11 @@ async def purge_csv_tables():
 
 @wrap
 async def cli_wrapper():
-    context["conn"] = await asyncpg.connect(dsn=config.DATABASE_URL)
+    context["conn"] = await asyncpg.connect(
```

Comment:
Indeed, maybe we should remove the wrapper as we now have `DATABASE_URL` & `DATABASE_URL_CSV` as connection strings, it can be confusing to wrap for `DATABASE_URL` by default.

Reply:
Let me know if [this solution](https://github.com/etalab/udata-hydra/pull/57/commits/099db57864d13300f2e259fdd4dbf0ad827e1aab) suits you :)

Reply:
Looking nice! Thanks for this